fcl/dunefd/detsim/standard_detsim_dune10kt_1x2x6_light.fcl

```diff
@@ -0,0 +1,17 @@
+#include "standard_detsim_dune10kt_1x2x6.fcl"
+
+physics:
+{
+    @table::physics
+    producers:
+    {
+      @table::physics.producers
+    }
+
+#    Set up of PDS only digitizer in 1x2x6 geo
+    simulate: 
+    [
+    opdigi,
+    rns
+    ]
+}
```

Comment:
I think this could be collapsed to 
`physics.simulate: [ opdigi,rns ]

Reply:
Yes, you are right of course

Reply:
Added a commit with your suggested changes